CHANGELOG.md

```diff
@@ -2,6 +2,32 @@
 
 This file is used to list changes made in each version of auditd.
 
+## 2.3.4 (2018-11-05)
+
+- Disable dokken testing in travis for now
+- Simplify ignore files
+- Reenable dokken testing on Ubuntu
+- Add Travis badge to readme
+- Fix travis config
+- kitchen yaml updated to include cis config. CIS erb updated to fix control 4.1.17
+- added OS version check for control 4.1.9
+- tested updates
```

Comment:
Not all these commits are worth documenting. I suspect these commits should have been rebased down to a lesser number of commits at the time they were PR'ed into the repo; I'm not sure if we can (or should try to) fix the commits retrospectively.